src/DI/TranslationExtension.php

```diff
@@ -78,7 +78,7 @@ public function getConfigSchema(): Schema
 				'vary' => Expect::array()->default([]),
 			]),
 			'translatorFactory' => Expect::string()->default(null),
-			'returnOriginalMessage' => Expect::bool()->default(false),
+			'returnOriginalMessage' => Expect::bool()->default(true),
```

Comment:
Breaking change.

Reply:
yes, I mention this in release notese here
https://github.com/contributte/translation/releases/tag/v0.9.0